parsimonious/tests/test_nodes.py

```diff
@@ -142,3 +142,8 @@ def visit_greeting(self, thing, visited_children):
             raise PrimalScream('This should percolate up!')
 
     assert_raises(PrimalScream, Screamer().parse, 'howdy')
+
+
+def test_node_inequality():
+    node = Node('text', 'o hai', 0, 5)
+    assert node != 5
```

Comment:
Absolutely! I am mostly familiar with pytest and not much with nose, so I just hacked away at it until I had a failing test.